pages/[[server]]/@[account]/[status].vue

```diff
@@ -96,6 +102,7 @@ onReactivated(() => {
               v-slot="{ item, index, active }"
               :items="context?.descendants || []"
               :min-item-size="200"
+              :buffer="clientHeight"
```

Comment:
Great stuff, this was affecting the UX a lot.

About using `clientHeight`, from [the docs](https://github.com/Akryum/vue-virtual-scroller/blob/master/packages/vue-virtual-scroller/README.md#props):
- buffer (default: 200): amount of pixel to add to edges of the scrolling visible area to start rendering items further away.

I think we should instead use the maximum post height. At this point, this is hard to know because we haven't yet implemented truncation of long posts but once that is in, we should be able to have a reasonable estimate. The issue I see with using the `clientHeight` is that it may be too big in some cases, or too small in others (or change after the component was setup and we aren't changing it). 

So maybe using a fixed value like 800 is better here?